usefulUtilities/soundEmitter/soundEmitter.js

```diff
@@ -111,6 +111,13 @@
         },
 
 
```

Comment:
Not a big one or blocker, but I noticed this whole file doesn't have very many comments.  Just bringing it to your attention in case you want to add them.